tests/test_ophys/test_tools_roiextractors.py

```diff
@@ -246,6 +246,48 @@ def test_add_two_imaging_planes(self):
         assert second_imaging_plane.name == second_imaging_plane_name
         assert second_imaging_plane.description == second_imaging_plane_description
 
+    def test_add_imaging_plane_raises_when_name_not_found_in_metadata(self):
+        """Test adding an imaging plane raises an error when the name is not found in the metadata."""
+        imaging_plane_name = "test_imaging_plane"
+        with self.assertRaisesWith(
+            exc_type=ValueError, exc_msg=f"Imaging plane '{imaging_plane_name}' not found in metadata."
+        ):
+            add_imaging_plane(nwbfile=self.nwbfile, metadata=self.metadata, imaging_plane_name=imaging_plane_name)
+
+    def test_add_two_imaging_planes_from_metadata(self):
```

Comment:
Nice.

We have:
* Test for adding a name when the name is not specified in the metadata
* Test for adding more than one ImagingPlane and distinguish them by name.

Maybe we can add one where the imaging plane is added by name but the name already exists on the nwbfile? Would that make sense?

Reply:
Yes that could make sense, I'll add it and thanks for the comments!

Reply:
it's always good to check, and thanks! :)